planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -105,19 +105,23 @@ bool isInLane(
 
 bool isInParkingLot(
   const std::shared_ptr<lanelet::LaneletMap> & lanelet_map_ptr,
-  const geometry_msgs::msg::Pose & current_pose)
+  const geometry_msgs::msg::Pose & current_pose,
+  const geometry_msgs::msg::Pose & goal_pose)
 {
-  const auto & p = current_pose.position;
-  const lanelet::Point3d search_point(lanelet::InvalId, p.x, p.y, p.z);
+  const auto & p_c = current_pose.position;
+  const auto & p_g = current_pose.position;
```

Comment:
Is this `goal_pose`? :thinking: 

```suggestion
  const auto & p_g = goal_pose.position;
```

Reply:
Thanks. I modified that part.